### PR TITLE
Announcements Tablets (Color Coded!!!!)

### PR DESCRIPTION
**Pull request by KingSuperBOB**
## About The Pull Request
Announcement tablets at the request of Kirie given to L2-LTs on the W-Corp mode and to Agent Captains in the LC13 mode.

## Why It's Good For The Game
Allows for loud and easy communication to these roles granting them better use than before. (Please please dont spam it)

## Changelog
:cl:
add: Added a description to the R-Corp announcement tablet
add: Added a W-Corp announcement tablet to L2 Recon
add: Added a L-Corp announcement tablet to Agent Captains
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/agent.dm [hunk 1]
```diff
@@ -142,6 +142,7 @@
 	head = /obj/item/clothing/head/hos/beret
 	ears = /obj/item/radio/headset/heads/agent_captain/alt
 	l_pocket = /obj/item/commandprojector
+	r_pocket = /obj/item/announcementmaker/lcorp
```
Comment (Kitsunemitsu) [inline code/modules/jobs/job_types/agent.dm:145]: They don't actually fit in pockets (I know it's lame)

Comment (KingSuperBOB) [reply code/modules/jobs/job_types/agent.dm:145]: Guh

File: ModularTegustation/tegu_items/rcorp/gadgets.dm [hunk 2]
```diff
@@ -37,10 +37,22 @@
...
+//Tablet variants
+/obj/item/announcementmaker/wcorp
+	name = "w-corp announcement tablet"
+	desc = "A special tablet used by WARP Clean up L2-LT agents to make rapid announcements on the battlefield."
+	icon_state = "tablet-blue"
+
+/obj/item/announcementmaker/lcorp
+	name = "l-corp announcement tablet"
+	desc = "A special tablet used by L-Corp Agent Captains to make rapid announcements during ongoing breaches and meltdowns."
+	icon_state = "tablet-brown"
```
Comment (Kitsunemitsu) [inline ModularTegustation/tegu_items/rcorp/gadgets.dm:58]: Forgot an extra line here

**Review by Kitsunemitsu — APPROVED** [review]
Approved. I think that this is a great addition to the game.

**Pull request merged**